### ISSUE TITLE
Adding import {FirebaseClientStorage} from @canner/storage throws error

### ISSUE THREAD

**Issue by odifek** [CannerCMS/cannercms#129]
> ERROR  Failed to compile with 1 errors                                 15:14:21
> 
> This dependency was not found:
> 
> * @canner/storage in ./canner.schema.js
> 
> To install it, you can run: npm install --save @canner/storage
> 

I have tried severally to no avail. I've also installed @canner/storage as recommended but not working

**Comment by abz53378** [comment]
Hi, @odifek , I'm not sure what caused this, so could you confirm the `@canner/storage` is indeed in the `node_modules` of your project?

**Comment by odifek** [comment]
Wow! So I discovered that node_modules are not being installed in the project folder. SO I had to do `npm init` first before installing @canner/storage. With that , the cms started working. Thank you guys for this amazing project.
I am a mobile app developer, and some of my personal projects need a backend, but because I am not yet fluent enough in Javascript to write my own backend, I needed a simpler way to create cms for a mobile app. Lo and behold, Canner.io is here. I appreciate guys.
